src/app/zap-templates/zcl/data-model/chip/channel-cluster.xml

```diff
@@ -62,20 +62,20 @@ limitations under the License.
     <item name="affiliateCallSign" type="CHAR_STRING" length="32"/>
   </struct>
 
-  <struct name="ChannelLineupInfo">
+  <struct name="LineupInfo">
     <cluster code="0x0504"/>
     <item name="operatorName" type="CHAR_STRING"/>
     <item name="lineupName" type="CHAR_STRING"/>
     <item name="postalCode" type="CHAR_STRING"/>
-    <item name="lineupInfoType" type="ChannelLineupInfoType"/>
+    <item name="lineupInfoTypeEnum" type="LineupInfoTypeEnum"/>
```

Comment:
This is called "LineupInfoType" in the spec.

Reply:
Updated.